package.json

```diff
@@ -13,7 +13,7 @@
     "!playwright-report/"
   ],
   "scripts": {
-    "e2e-tests-pro": "npx playwright test",
+    "e2e-tests-pro": "npx playwright test erpnext-openmrs",
```

Comment:
`erpnext-openmrs` should be part of Ozone FOSS E2E tests(`e2e-tests-foss"`).

Reply:
Yes. I'm specifically testing out O3 → ERPNext flows and then will update the scripts accordingly.